README.md

```diff
@@ -15,7 +15,7 @@ included.
 ## Install
 
 ```
-npm install @roamhq/wrtc
+npm install @avahq/wrtc
```

Comment:
@roamhq


You have a few places where you have avahq instead it should be roamhq